Server/VikingVault.Services/PDFGeneratorService.cs

```diff
@@ -81,30 +81,65 @@ public void AddTransactionListIntoPDF(PdfPage page, string userId, string timeFi
+            graphics.DrawString("Currency", columnHeaderFont, columnHeaderBrush, new PointF(195, 150));
+            graphics.DrawString("Details", columnHeaderFont, columnHeaderBrush, new PointF(256, 150));
+            graphics.DrawString("Description", columnHeaderFont, columnHeaderBrush, new PointF(407, 150));
 
-            int x = 40;
+            int x = 15;
             int y = 180;
 
             foreach (TransactionDTO transaction in transactionList)
             {
-                graphics.DrawString(transaction.Date.ToShortDateString().ToString(), columnDataFont, columnDataBrush, new PointF(x, y));
-                graphics.DrawString(transaction.Currency.ToString(), columnDataFont, columnDataBrush, new PointF(x + 120, y));
-                graphics.DrawString(transaction.Amount.ToString(), columnDataFont, columnDataBrush, new PointF(x + 230, y));
-                graphics.DrawString(transaction.Details.ToString(), columnDataFont, columnDataBrush, new PointF(x + 360, y));
+                graphics.DrawString(transaction.Date.ToShortDateString(), columnDataFont, columnDataBrush, new PointF(x, y));
+                graphics.DrawString(transaction.Type, columnDataFont, columnDataBrush, new PointF(x + 60, y));
+                graphics.DrawString(transaction.Amount.ToString().Trim('-'), columnDataFont, columnDataBrush, new PointF(x + 110, y));
+                graphics.DrawString(transaction.Currency.ToUpper(), columnDataFont, columnDataBrush, new PointF(x + 180, y));
+
+                if (transaction.Type == "Exchange")
+                {
+                    graphics.DrawString("Exchanged into " + transaction.Details.ToString(), columnDataFont, columnDataBrush, new PointF(x + 240, y));
+                    graphics.DrawString("Money Exchange", columnDataFont, columnDataBrush, new PointF(x + 390, y));
+                }
+                else
+                {
+                    string transferText = "";
+                    string transactionDetails = TrimmTransactionDetails(transaction);
+
+                    if (transaction.IsUserSender == true)
+                    {
+                        transferText = "Sent to " + transaction.Receiver.FirstName.ToString() + " " + transaction.Receiver.LastName.ToString();
+                    }
+                    else
+                    {
+                        transferText = "Received from " + transaction.Sender.FirstName.ToString() + " " + transaction.Sender.LastName.ToString();
+                    }
+
+                    graphics.DrawString(transferText, columnDataFont, columnDataBrush, new PointF(x + 240, y));
+                    graphics.DrawString(transactionDetails, columnDataFont, columnDataBrush, new PointF(x + 390, y));
+                }
                 y = y + 25;
             }
         }
 
+        public string TrimmTransactionDetails (TransactionDTO transaction)
+        {
+            string transactionDetails = transaction.Details.ToString();
```

Comment:
I would use **var** any time the initialization of the variable clearly tells the type. For Example here you call ToString() method, it is clear that this is a string

Reply:
Changed it.